Tests/ActionTests/ActionTests.swift

```diff
@@ -107,479 +107,479 @@ class ActionTests: QuickSpec {
 					}
-					
+
 					it("errors observable receives generated errors") {
 						XCTAssertEqual(errors.events, [
-							next(10, .notEnabled),
-							next(20, .notEnabled),
+							Recorded.next(10, .notEnabled),
+							Recorded.next(20, .notEnabled)
 							])
 					}
-                    
+
                     it("underlyingError observable receives zero generated errors") {
                         XCTAssertEqual(underlyingError.events.count, 0)
                     }
-					
+
 					it("disabled") {
 						XCTAssertEqual(enabled.events, [
-							next(0, false),
+							Recorded.next(0, false)
 							])
 					}
-					
+
 					it("never be executing") {
 						XCTAssertEqual(executing.events, [
-							next(0, false),
+							Recorded.next(0, false)
 							])
 					}
-					
+
 					it("never executes") {
 						expect(executionObservables.events).to(beEmpty())
 					}
 				}
-				
+
 				var action: Action<String, String>!
-				
+
 				beforeEach {
 					action = Action(enabledIf: Observable.just(false)) { Observable.just($0) }
 					bindAction(action: action)
 				}
-				
+
 				context("trigger via inputs subject") {
 					beforeEach {
 						scheduler.scheduleAt(10) { action.inputs.onNext("a") }
 						scheduler.scheduleAt(20) { action.inputs.onNext("b") }
 						scheduler.start()
 					}
-					
+
 					itBehavesLike("send notEnabled errors to errors observable")
 				}
-				
+
 				context("trigger via execute() method") {
 					beforeEach {
 						scheduler.scheduleAt(10) { action.execute("a") }
 						scheduler.scheduleAt(20) { action.execute("b") }
 						scheduler.start()
 					}
-					
+
 					itBehavesLike("send notEnabled errors to errors observable")
 				}
 			}
 		}
-		
+
 		describe("execute function return value") {
 			var action: Action<String, String>!
 			var element: TestableObserver<String>!
 			var executionObservables: TestableObserver<Observable<String>>!
-			
+
 			beforeEach {
 				element = scheduler.createObserver(String.self)
 				executionObservables = scheduler.createObserver(Observable<String>.self)
 			}
-			
+
 			func bindAndExecuteTwice(action: Action<String, String>) {
 				action.executionObservables
 					.bind(to: executionObservables)
 					.disposed(by: disposeBag)
-				
+
 				scheduler.scheduleAt(10) {
 					action.execute("a")
 						.bind(to: element)
 						.disposed(by: disposeBag)
 				}
-				
+
 				scheduler.scheduleAt(20) {
 					action.execute("b")
 						.bind(to: element)
 						.disposed(by: disposeBag)
 				}
-				
+
 				scheduler.start()
 			}
-			
+
 			context("single element action") {
 				beforeEach {
 					action = Action { Observable.just($0) }
 					bindAndExecuteTwice(action: action)
 				}
-				
+
 				it("element receives single value for each execution") {
 					XCTAssertEqual(element.events, [
-						next(10, "a"),
-						completed(10),
-						next(20, "b"),
-						completed(20),
+						Recorded.next(10, "a"),
+                        Recorded.completed(10),
```

Comment:
misaligned

Reply:
Thought swiftlint autocorrect would fix it :(